Aftonbladet.js

```diff
@@ -0,0 +1,209 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://ww*\.aftonbladet\.se",
```

Comment:
The regexp looks strange here. It has to be one or many `w`s? Moreover, the `.` have to escaped (once in Scaffold which will result in twice here). I suggest to do write in Scaffold:
```
^https?://(www\.)?aftonbladet\.se
```
which will result here into
```
^https?://(www\\.)?aftonbladet\\.se
```